codex-rs/core/Cargo.toml

```diff
@@ -44,10 +39,10 @@ tokio = { version = "1", features = [
     "signal",
 ] }
 tokio-util = "0.7.14"
-toml = "0.8.20"
+toml = "0.9.0"
 tracing = { version = "0.1.41", features = ["log"] }
 tree-sitter = "0.25.3"
-tree-sitter-bash = "0.23.3"
+tree-sitter-bash = "0.25.0"
```

Comment:
Admittedly, I'm not sure how well everything is covered by unit tests, so auto-updating third party deps automatically is a little dicey for us right now.